src/assets/GoogleIcon.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default ({ active }) => (
+  <div style={{ marginRight: 10, background: active ? '#eee' : '#fff', padding: 10, borderRadius: 2 }}>
```

Comment:
this is cool, I didn't know there was an inline svg version of the google icon that we should be using